### PR TITLE
Change all the required types from str to byte array

### PR DESCRIPTION
**Pull request by Eitu33**
⚠️ DO NOT merge this until corresponding updates have been done on the other repositories

### CODE REVIEW

File: assembly/std/index.ts [hunk 1]
```diff
@@ -32,8 +32,10 @@ export function call(
   functionName: string,
   args: Args,
   coins: u64,
-): string {
-  return env.call(at.toByteString(), functionName, args.serialize(), coins);
+): StaticArray<u8> {
+  // IMPORTANT TODO: change this after `Args` update
```
Comment (sydhds) [inline assembly/std/index.ts:36]: Args feature is now ok no?

Comment (Eitu33) [reply assembly/std/index.ts:36]: Yes but it does not serialize into the right data type, need to sync with the inno team on that to see if they want me to take care of it or if they'd rather do it themselves

Comment (AurelienFT) [reply assembly/std/index.ts]: Done.

**Comment by AurelienFT** [comment]
@ASAPSegfault and @peterjah I have put you as a review because you are aware of the situation but if you want feel free to switch it to someone else.

**Review by peterjah — APPROVED** [review]

**Pull request merged**